jni/events/baseaudioevent.h

```diff
@@ -75,6 +78,11 @@ class BaseAudioEvent
         virtual BaseInstrument* getInstrument(); // retrieve reference to the instrument this event belongs to
         virtual void setInstrument( BaseInstrument* aInstrument ); // set / swap instrument this event belongs to
 
+        virtual void play(); // enable the event and play it back immediately (e.g. "noteOn" for instant playback)
+        virtual void stop(); // immediately stops playing the live-auditioned event (e.g. "noteOff")
+
```

Comment:
play/stop were previously only available to SampleEvent which is silly, make it a BaseAudioEvent feature (SynthEvents for instance would be playing back live when using a specific constructor, which is silly).